ruby/deepgram-transcribe-video/README.md

```diff
@@ -0,0 +1,67 @@
+# 🔉📃 Transcribe Video with Deepgram
+
+A Ruby Cloud Function for transcribing video using [Deepgram API](https://deepgram.com/).
+
+_Example input:_
+
+```json
+{
+    "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Example output:_
+
+
+```json
+{
+    "success": "true",
+    "deepgramData": "another big problem in the speech analytics space. When customers first bring the software on. Is that they they are blown away by the fact that an engine can monitor hundreds of Kpis. Right? Everything from my..."
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+- **DEEPGRAM_API_KEY** - Deepgram API Key
+
+ℹ️ _Create your Deepgram API key at console.deepgram.com_
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
```

Comment:
```suggestion
git clone https://github.com/open-runtimes/examples.git && cd examples
```

I would suggest not having the `$` here because it makes it harder to copy and paste.